core/forms.py

```diff
@@ -0,0 +1,15 @@
+from django import forms
+from django.contrib.auth.forms import UserCreationForm, UsernameField
+from .models import User
+
+class CustomUserCreationForm(UserCreationForm):
+    username = UsernameField(label='Nazwa użytkownika', help_text='', widget=forms.TextInput)
+    email = forms.EmailField(label='Email',help_text='', widget=forms.EmailInput)
+    password1 = forms.CharField(label='Hasło', help_text='', widget=forms.PasswordInput)
+    password2 = forms.CharField(label='Powtórz hasło', help_text='', widget=forms.PasswordInput)
+
+
+    class Meta:
+        model = User
+        fields = ('username', 'email')
+        fields_classes = {'username' : UsernameField, 'email' : forms.EmailField}
```

Comment:
No dobra, weź sobie ogarnij jakiegoś formattera do kodu, chcesz, to mogę wam zintegrować z repozytorium blacka, przez co z każdym commitem będzie formatowało cały kod.